luigi_pipeline/seqr_loading.py

```diff
@@ -200,14 +200,20 @@ def validate_mt(mt, genome_version, sample_type):
             if sample_type != 'WES':
                 raise SeqrValidationError(
                     'Sample type validation error: dataset sample-type is specified as {} but appears to be '
-                    'WGS because it contains many common coding variants'.format(sample_type)
+                    '{} because it contains many common coding variants'.format(
+                        "WGS",
+                        "WES",
```

Comment:
Thoughts on just filling in the data type and making this a normal string we don't need to format? Not sure if there is a best practice around errors and formatting string but feel a basic string "...sample-type is specified as WGS  but appears to be WES...." is more readable?

Reply:
No real reason as is!  I think ideally we'd have one error function that would take the sample types as arguments (which is what I attempted and then backtracked to this...) but just plopping in the strings is fine!